web/html/src/manager/clusters/remove-node/remove-node.js

```diff
@@ -0,0 +1,88 @@
+// @flow
+import {hot} from 'react-hot-loader';
+import React, {useState}  from 'react';
+import {TopPanel} from "components/panels/TopPanel";
+import {HashRouter, Route, Switch} from 'components/utils/HashRouter';
+import withPageWrapper from 'components/general/with-page-wrapper';
+import useClustersApi, {withErrorMessages} from '../shared/api/use-clusters-api';
+import ScheduleClusterAction from '../shared/ui/schedule-cluster-action';
+import {SystemLink} from 'components/links';
+import FormulaConfig from '../shared/ui/formula-config';
+
+import type {ClusterType, FormulaValuesType, ClusterNodeType} from '../shared/api/use-clusters-api';
+
+type Props = {
+  cluster: ClusterType,
+  nodes: Array<ClusterNodeType>,
+  serverId: ?number,
+  flashMessage: string,
+};
+
+const RemoveNode = (props: Props) => {
+    const [removeConfig, setRemoveConfig] = useState<?FormulaValuesType>(null);
+    const {scheduleRemoveNode} = useClustersApi();
+
+    const scheduleRemove = (earliest: Date, actionChain: ?string): Promise<any> => {
+        if (removeConfig) {
+            return scheduleRemoveNode(props.cluster.id,
+                 props.nodes
+                    .filter(node => node.server ? true : false)
+                    .map(node => node.server ? node.server.id : 0),
```

Comment:
I think this is enough after the `.filter()` call
```suggestion
                    .map(node => node.server.id),
```

Reply:
The reason for this is Flowtype which is too strict in this case.